src/askscript/parser/askscript.grammar.pegjs

```diff
@@ -223,6 +220,11 @@ nonEmptyTypeList =
     ws* t:type ws* ',' tL:nonEmptyTypeList { return tL.unshift(t), tL }
   / ws* t:type { return [t] }
 
+// FIXME:
+// - types should allow any functional expression, because we can define our own types
+// - function type
+// - fix src/askscript/__tests__/00-documentation-examples/documentation24-function_type.ask
+// - fix src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.ask
 type = 
```

Comment:
@czerwinskilukasz1 - type grammar needs to be more flexible - we can do it in a follow up

Reply:
@mhagmajer Let's keep PRs small and do it in a follow up.